.github/workflows/master.yml

```diff
@@ -403,6 +403,9 @@ jobs:
           minikube start --cpus 2 --memory 4096 --kubernetes-version=${KUBERNETES_VERSION} --force
           # https://minikube.sigs.k8s.io/docs/handbook/pushing/#7-loading-directly-to-in-cluster-container-runtime
           minikube image load apache/kyuubi:latest
+          # pre-install spark into minikube
+          docker pull apache/spark:3.4.1
```

Comment:
cachable?

Reply:
I can't found how to cache pulled docker image.

Reply:
then how does it help? just perform the pulling action slightly in advance?

Reply:
alright, we educated that `kubectl docker` has an unstable network even though we don't know why.